src/settings/MappingProfiles/detailsSections/utils.js

```diff
@@ -242,7 +242,7 @@ export const getAccountNumbersByCode = organization => {
 };
 
 export const renderAmountValue = (amountValue, amountType, currency) => {
-  const amountSymbol = amountType === '"percentage"' ?
+  const amountSymbol = amountType === '"Percentage"' ?
```

Comment:
let's move `Percentage` and `Amoubt` this to constants
```suggestion
  const amountSymbol = amountType === `"${PERCENTAGE_VALUE}"` ?
```